miden-lib/asm/kernels/transaction/api.masm

```diff
@@ -70,11 +73,14 @@ end
 
 #! Returns the account id.
 #!
-#! Stack: [0]
+#! Stack: [KERNEL_PROCEDURE_HASH, 0]
 #! Output: [acct_id]
 #!
 #! - acct_id is the account id.
 export.get_account_id
```

Comment:
I did it at first, but in that case hashes of these (`proc`) functions are not stored neither in `Kernel`, nor in `ModuleInfo` procedures (if I get it right, they are just got inlined at the places where they are used), so looks like the only easy way to obtain their hashes and be able to call them is to leave them as `export` functions.

Reply:
Hmmm - good point. I think if they are not exported they are probably considered "dead code" and would never be added to the `MastForest`. It may be a good idea to add ability to annotate procedures to make sure they are always included in the `MastForest` (something like `allow_dead_code`). Could you mention this in https://github.com/0xPolygonMiden/miden-vm/issues/1434 ?

Reply:
Also, we should create an issue to change `export` to `proc` in the future.